recipe/meta.yaml

```diff
@@ -12,6 +12,7 @@ source:
 build:
   noarch: python
```

Comment:
If we will build an **arch-specific** package then `noarch: python` should be removed.